src/EFCore/ModelBuilder.cs

```diff
@@ -142,6 +157,22 @@ public virtual EntityTypeBuilder Entity([NotNull] string name)
             return new EntityTypeBuilder(Builder.Entity(name, ConfigurationSource.Explicit).Metadata);
         }
 
+        /// <summary>
+        ///     Returns an object that can be used to configure a given entity type in the model.
+        ///     If an entity type with the provided name is not already part of the model,
+        ///     a new entity type that does not have a corresponding CLR type will be added to the model.
```

Comment:
Not sure why this is. If no entity type has that name shouldn't you create a new entity type with CLR Type = clrType? Or is this a typo?

```suggestion
        ///     a new entity type with the given CLR type will be added to the model.
```